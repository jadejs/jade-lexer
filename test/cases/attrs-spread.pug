div(...attrs)
div(a='bleh', ...attrs)
div(a='bleh' ...attrs)
div(a ...attrs)
div(a ... attrs)
div(...!attrs)
div(... 
!
	attrs)
div("...attrs" ...attrs, '...attrs' ='val' "...")
