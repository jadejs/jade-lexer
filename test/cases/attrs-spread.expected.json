{"type":"tag","line":1,"col":1,"val":"div"}
{"type":"start-attributes","line":1,"col":4}
{"type":"spread-attribute","line":1,"col":5,"val":"attrs","mustEscape":true}
{"type":"end-attributes","line":1,"col":13}
{"type":"newline","line":2,"col":1}
{"type":"tag","line":2,"col":1,"val":"div"}
{"type":"start-attributes","line":2,"col":4}
{"type":"attribute","line":2,"col":5,"name":"a","val":"'bleh'","mustEscape":true}
{"type":"spread-attribute","line":2,"col":15,"val":"attrs","mustEscape":true}
{"type":"end-attributes","line":2,"col":23}
{"type":"newline","line":3,"col":1}
{"type":"tag","line":3,"col":1,"val":"div"}
{"type":"start-attributes","line":3,"col":4}
{"type":"attribute","line":3,"col":5,"name":"a","val":"'bleh'","mustEscape":true}
{"type":"spread-attribute","line":3,"col":14,"val":"attrs","mustEscape":true}
{"type":"end-attributes","line":3,"col":22}
{"type":"newline","line":4,"col":1}
{"type":"tag","line":4,"col":1,"val":"div"}
{"type":"start-attributes","line":4,"col":4}
{"type":"attribute","line":4,"col":5,"name":"a","val":true,"mustEscape":true}
{"type":"spread-attribute","line":4,"col":7,"val":"attrs","mustEscape":true}
{"type":"end-attributes","line":4,"col":15}
{"type":"newline","line":5,"col":1}
{"type":"tag","line":5,"col":1,"val":"div"}
{"type":"start-attributes","line":5,"col":4}
{"type":"attribute","line":5,"col":5,"name":"a","val":true,"mustEscape":true}
{"type":"spread-attribute","line":5,"col":7,"val":"attrs","mustEscape":true}
{"type":"end-attributes","line":5,"col":16}
{"type":"newline","line":6,"col":1}
{"type":"tag","line":6,"col":1,"val":"div"}
{"type":"start-attributes","line":6,"col":4}
{"type":"spread-attribute","line":6,"col":5,"val":"attrs","mustEscape":false}
{"type":"end-attributes","line":6,"col":14}
{"type":"newline","line":7,"col":1}
{"type":"tag","line":7,"col":1,"val":"div"}
{"type":"start-attributes","line":7,"col":4}
{"type":"spread-attribute","line":7,"col":5,"val":"attrs","mustEscape":false}
{"type":"end-attributes","line":9,"col":7}
{"type":"newline","line":10,"col":1}
{"type":"tag","line":10,"col":1,"val":"div"}
{"type":"start-attributes","line":10,"col":4}
{"type":"attribute","line":10,"col":5,"name":"...attrs","val":true,"mustEscape":true}
{"type":"spread-attribute","line":10,"col":16,"val":"attrs","mustEscape":true}
{"type":"attribute","line":10,"col":26,"name":"...attrs","val":"'val'","mustEscape":true}
{"type":"attribute","line":10,"col":44,"name":"...","val":true,"mustEscape":true}
{"type":"end-attributes","line":10,"col":49}
{"type":"newline","line":11,"col":1}
{"type":"eos","line":11,"col":1}